tasks/jscs.js

```diff
@@ -8,11 +8,12 @@ module.exports = function (grunt) {
   grunt.config('jscs', {
     src: [
       '**/*.js',
+      'tests/**/*.js',
       '!node_modules/**',
       '!build/**',
       '!components/**',
       '!docs/**',
-      '!tests/**'
+      '!tests/addons/**'
```

Comment:
I think we only want to disable linting on the _./tests/addons/sinon.js_ file specifically, not the entire directory. IIRC, the other files in that folder are our libs and not 3rd party.

Let's change this `src` glob to something a bit more specific, like this:
```js
src: [
  '{,client/**/,node/,tasks/,tests/**/}*.js'
  '!tests/addons/sinon.js'
]
```